ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiWorldStateUpdater.java

```diff
@@ -201,7 +182,7 @@ public void commit() {
                 (keyHash, entryValue) -> {
                   final Hash slotHash = Hash.wrap(keyHash);
                   if (!deletedStorageUpdates.containsKey(slotHash)) {
-                    final UInt256 value = UInt256.fromBytes(RLP.decodeOne(entryValue));
+                    final Bytes32 value = Bytes32.leftPad(Bytes.wrap(RLP.decodeOne(entryValue)));
```

Comment:
Why do we need the Bytes.wrap here?

Reply:
We don't.  Large scale change missed it.